examples/csv_read.rs

```diff
@@ -0,0 +1,36 @@
+use arrow2::error::Result;
+use arrow2::io::csv::read;
+use arrow2::record_batch::RecordBatch;
+
+fn read_path(path: &str, projection: Option<&[usize]>) -> Result<RecordBatch> {
```

Comment:
Silly question, What is projection?

Reply:
when we want to skip certain columns. The indexes represent the columns that we want. Should be documented ^_^